src/Panel/SqlLogPanel.php

```diff
@@ -79,7 +79,7 @@ public function data()
         return [
             'tables' => array_map(function (Table $table) {
                 return $table->getAlias();
-            }, TableRegistry::genericInstances()),
+            }, (new TableLocator)->genericInstances()),
```

Comment:
You need the global instance for this feature to work correctly

Reply:
use of `LocatorAwareTrait` ? maybe

Reply:
Yes!

Reply:
`LocatorAwareTrait::getTableLocator()` uses `TableRegistry::getTableLocator();` internally 
I changed to use `TableRegistry::getTableLocator();`